custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -1170,6 +1170,18 @@
           "type": "property_name",
           "property_name": "num_anc_complete"
         }
+     },
+     {
+        "transform": {},
+        "datatype": "small_integer",
+        "type": "expression",
+        "is_nullable": true,
+        "column_id": "child_name",
+        "expression": {
+          "datatype": null,
+          "type": "property_name",
+          "property_name": "child_name"
```

Comment:
How does this work for twins?

Reply:
Upon further look at the app (from the case summary), it looks like this is only filled when the child is registered in the ccs stage (from the add member form), not on normal delivery. That is also the only case where twins are registered with different ccs record cases, i believe. I have changed the join condition based on this info so that it is used as a tie breaker only when it exists, which should also be the only time we were seeing row duplication.